style-guide.md

```diff
@@ -0,0 +1,220 @@
+> 
+> - Hold a referendum on the relay chain...[shortened]
+> - Hold a vote between the stakeholders...
+> - Users can safely migrate all usage away from...
+> - Perform one of the fundraising methods...
+
+Items 1, 2, and 4 start with a verb. Item 3 starts with a noun. Put them into the same form.
+
+### American vs. British
+
+You should use American spellings for words, unless you are referring to a specific item (function, trait, module, etc.). This means:
+
+- Finalize over finalise
+- Behavior over behaviour
+
+When in doubt, refer to [ngram](https://books.google.com/ngrams/graph?content=finalize%2C+finalise&case_insensitive=on&year_start=1900&year_end=2018&corpus=15&smoothing=3&share=&direct_url=t4%3B%2Cfinalize%3B%2Cc0%3B%2Cs0%3B%3Bfinalize%3B%2Cc0%3B%3BFinalize%3B%2Cc0%3B%3BFINALIZE%3B%2Cc0%3B.t4%3B%2Cfinalise%3B%2Cc0%3B%2Cs0%3B%3Bfinalise%3B%2Cc0%3B%3BFinalise%3B%2Cc0).
+
+### Check Your Assumptions
+
+When writing reference documentation, refrain from making assumptions about how something will be used, especially with dispatchable and public functions.
+
+Statements like, "This function is usually used for..." make assumptions about what users will do. Even if it is accurate for our own code, its usage may change without being updated.
+
+## Common Grammar and Usage Mistakes
+
+### That vs. Which
+
+"That" and "which" are often used interchangeably in conversational English, but they have different meanings. That is restrictive: you cannot remove it without changing the meaning of the sentence.
+
+> Dispatchable functions that do not have an `origin` parameter in the function signature can only be called by `Root`.
+
+The clause "that do not have an `origin` parameter in the function signature" changes the meaning of the sentence. The sentence would not be true if you removed it.
+
+Which precedes a nonrestrictive clause. Removing it does not change the meaning of the sentence. A nonrestrictive clause is extra information.
+
+> Dispatchable functions, which are listed in the `Call` enum, always include an `origin` parameter after macro expansion.
+
+The clause "which are listed in the `Call` enum" does not change the meaning of the sentence, it only adds information.
+
+Example that can lead to confusion:
+
+> The sudo key can execute dispatchable functions which require a `Root` call.
+
+If you interpret this as nonrestrictive, then this sentence is saying that all dispatchable functions require a `Root` call, which is not true. It should say, "functions **that** require a `Root` call."
+
+Readers can often infer when a "which" clause is restrictive versus unrestrictive. If you want to be precise (you do), always use "that" for a restrictive clause.
+
+### Subject and Pronoun Disagreement
+
+Ensure that pronouns and possessives match the subject they are referring to.
+
+> Incorrect: The process of an account removing their entire holding of an asset.
+
+This is incorrect because "an account" is singular and "their" is plural.
```

Comment:
I thought that this is now acceptable and the politically correct thing to do...

Reply:
Sort of. The Washington Post allows singular they, but pretty much every other formal guide rejects it. Without taking sides in the political correctness debate, I think there's usually an elegant sentence reconstruction that avoids the issue entirely.

Reply:
Added another way to handle